docs/getting-started/how-to-publish-a-contract.md

```diff
@@ -122,9 +130,15 @@ The `scrypt.index.json` file will be generated at typescript compile time in the
 }
 ```
 
-The library user could just use `npm install` to install the dependency, and then use the exported smart contract library as a normal module in their smart contracts.
+You can publish the library on [NPM](https://www.npmjs.com/) by running the following command in the projects root directory:
+
+```sh
+scrypt publish
+```
+
+This will check the projects structure, build it and publish it. After the library is published, users can just import it in any other project just like regular NPM packages.
 
-**Please Note** that named imports is not supported yet. You should only use an import statement like this:
+**Please Note** that named imports are not supported yet. You should only use an import statement like this:
```

Comment:
You should only use an import statement like this:
->
You should only import like this: